libs/ui/src/components/NeoNftCard/NeoNftCard.vue

```diff
@@ -9,51 +9,52 @@
-      <div class="is-flex is-flex-direction-column">
-        <span class="is-ellipsis has-text-weight-bold">{{
-          nft.name || '--'
-        }}</span>
+      <div
+        class="nft-media-info is-flex is-flex-direction-column"
+        :class="`nft-media-info__${variant}`">
+        <div class="is-flex is-flex-direction-column">
+          <span class="is-ellipsis has-text-weight-bold">{{
+            nft.name || '--'
+          }}</span>
 
-        <CollectionDetailsPopover
-          v-if="
-            variant !== 'minimal' && (nft.collection.name || nft.collection.id)
-          "
-          :show-delay="collectionPopoverShowDelay"
-          :nft="nft"
-          class="is-ellipsis">
-          <template #trigger>
-            <nuxt-link
-              :title="nft.collectionName || nft.collection.name"
-              :to="`/${prefix}/collection/${nft.collection.id}`"
-              class="is-size-7 nft-info-collection-name">
-              {{ nft.collection.name || '--' }}
-            </nuxt-link>
-          </template>
-        </CollectionDetailsPopover>
-      </div>
+          <CollectionDetailsPopover
+            v-if="
+              variant !== 'minimal' &&
+              (nft.collection.name || nft.collection.id)
+            "
+            :show-delay="collectionPopoverShowDelay"
+            :nft="nft"
+            class="is-ellipsis">
+            <template #trigger>
+              <nuxt-link
+                :title="nft.collectionName || nft.collection.name"
+                :to="`/${prefix}/collection/${nft.collection.id}`"
+                class="is-size-7 nft-info-collection-name">
+                {{ nft.collection.name || '--' }}
+              </nuxt-link>
+            </template>
+          </CollectionDetailsPopover>
+        </div>
 
-      <div
-        class="is-flex is-align-items-center mt-2 is-ellipsis nft-media-info-footer"
-        :class="[
-          showPrice
-            ? 'is-justify-content-space-between'
-            : 'is-justify-content-end',
-        ]">
-        <CommonTokenMoney
-          v-if="showPrice"
-          :value="nft.price"
-          data-cy="card-money" />
-        <span
-          v-if="variant !== 'minimal'"
-          class="chain-name is-capitalized is-size-7"
-          >{{ getChainNameByPrefix(prefix) }}</span
-        >
+        <div
+          class="is-flex is-align-items-center mt-2 is-ellipsis nft-media-info-footer"
+          :class="[
+            showPrice
+              ? 'is-justify-content-space-between'
+              : 'is-justify-content-end',
+          ]">
+          <CommonTokenMoney
+            v-if="showPrice"
+            :value="nft.price"
+            data-cy="card-money" />
+          <span
+            v-if="variant !== 'minimal'"
+            class="chain-name is-capitalized is-size-7"
+            >{{ getChainNameByPrefix(prefix) }}</span
+          >
+        </div>
       </div>
-    </div>
+    </nuxt-link>
```

Comment:
```suggestion
    </nuxt-link>
```
We are not using `nuxt` in `libs/ui`

Reply:
it was already there though and seems to work, I replaced it with <a> though now